core/primitives/src/views.rs

```diff
@@ -928,6 +935,33 @@ impl fmt::Debug for FinalExecutionOutcomeView {
     }
 }
 
+/// Final execution outcome of the transaction and all of subsequent the receipts. Also includes
+/// the generated receipt.
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, PartialEq, Eq, Clone)]
+pub struct FinalExecutionOutcomeWithReceiptView {
+    /// Execution status. Contains the result in case of successful execution.
+    pub status: FinalExecutionStatus,
+    /// Signed Transaction
+    pub transaction: SignedTransactionView,
+    /// The execution outcome of the signed transaction.
+    pub transaction_outcome: ExecutionOutcomeWithIdView,
+    /// The execution outcome of receipts.
+    pub receipts_outcome: Vec<ExecutionOutcomeWithIdView>,
+    /// Receipts generated from the transaction
+    pub receipts: Vec<ReceiptView>,
+}
```

Comment:
Instead of copy-pasting the fields, I suggest we compose `FinalExecutionOutcomeWithReceiptView` of inner `FinalExecutionOutcomeView` and `Vec<ReceiptView>`. And thus you can easily get `FinalExecutionOutcomeView` out of `FinalExecutionOutcomeWithReceiptView`.